<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PBCore Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="descMD">
     <attribute name="title">
      <string>Descriptive Metadata</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>431</width>
        <height>23</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="resourceIdForm">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>59</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Resource ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="resource_id">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>431</width>
        <height>23</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="resourceIdForm_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>59</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Resource Title:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="resource_id_2">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>338</width>
        <height>200</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Language(s):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="language_list"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QToolButton" name="toolButton_6">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_5">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="languagelist_upbutton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>↑</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="languagelist_downbutton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>↓</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>301</width>
        <height>200</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Description(s):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="description_list"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_2">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="descriptionlist_upbutton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>↑</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="descriptionlist_downbutton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>↓</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>90</y>
        <width>338</width>
        <height>199</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Contributor(s):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="contributor_list"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QToolButton" name="toolButton_4">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_3">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="contributorlist_upbutton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>↑</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="contributorlist_downbutton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>↓</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="analogInst">
     <attribute name="title">
      <string>Analog Instantiation</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>331</width>
        <height>49</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Identifier:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="analog_id"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Source:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="analog_source"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>351</width>
        <height>49</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Location:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="analog_location"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Date:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="analog_date"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QComboBox" name="tape_type">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1/4&quot; audio tape</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cassette tape</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Microcassette</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="analog_tracklayout">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Half-track stereo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Half-track mono</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Full-track mono</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Quarter-track stereo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Four-track</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>70</y>
        <width>411</width>
        <height>551</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>PREMIS event(s):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="analog_premis_list"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QToolButton" name="toolButton_8">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_7">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="analogpremis_upbutton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>↑</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="analogpremis_downbutton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>↓</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QComboBox" name="analog_generation">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Original</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Copy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="analog_tapespeed">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>7.5 ips</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3.75 ips</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1.875 ips</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15 ips</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <zorder></zorder>
     <zorder></zorder>
     <zorder></zorder>
     <zorder></zorder>
     <zorder>tape_type</zorder>
     <zorder>analog_tracklayout</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>analog_generation</zorder>
     <zorder>analog_tapespeed</zorder>
    </widget>
    <widget class="QWidget" name="digitalInst">
     <attribute name="title">
      <string>Digital Instantiations</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>701</width>
        <height>601</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_16">
         <item>
          <widget class="QListWidget" name="digitalinstantiation_list"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QToolButton" name="toolButton_10">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_9">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="digitalinstantiation_upbutton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>↑</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="digitalinstantiation_downbutton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>↓</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <zorder>layoutWidget_7</zorder>
     <zorder>digitalinstantiation_list</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionBlow_away">
   <property name="text">
    <string>Blow away...</string>
   </property>
  </action>
  <action name="actionFoot">
   <property name="text">
    <string>Foot</string>
   </property>
  </action>
  <action name="actionHead">
   <property name="text">
    <string>Head</string>
   </property>
  </action>
  <action name="actionFingers">
   <property name="text">
    <string>Fingers</string>
   </property>
  </action>
  <action name="actionFoo">
   <property name="text">
    <string>Foo</string>
   </property>
  </action>
  <action name="actionBar">
   <property name="text">
    <string>Bar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
